src/frontend/src/common/components/BaseButton.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <button
+    v-on="$listeners"
+    :disabled="disabled"
+    :type="type"
+    :class="getClasses"
+  >
+    <slot></slot>
```

Comment:
```suggestion
    class="button"
    :class="{'button--disabled': disabled}"
  >
```
Лучше делать таким образом, чтобы не разводить кучу computed свойств